osu.Android/monogc.txt

```diff
@@ -0,0 +1 @@
+﻿MONO_GC_PARAMS=nursery-size=8m
```

Comment:
any explanation on where this came from? and if it is actually being used?

Reply:
That was for the ImageSharp problem, which didn't work. I'll have to get rid of that file, since we don't need it.